types/src/v17/mod.rs

```diff
@@ -137,11 +137,11 @@
 //!
 //! | JSON-PRC Method Name               | Returns         | Notes                                  |
 //! |:-----------------------------------|:---------------:|:--------------------------------------:|
-//! | createmultisig                     | version + model | TODO                                   |
-//! | estimatesmartfee                   | returns nothing | TODO                                   |
-//! | signmessagewithprivkey             | returns string  | TODO                                   |
-//! | validateaddress                    | version + model | TODO                                   |
-//! | verifymessage                      | returns boolean | TODO                                   |
+//! | createmultisig                     | version + model |                                        |
+//! | estimatesmartfee                   | returns nothing |                                        |
+//! | signmessagewithprivkey             | returns string  |                                        |
+//! | validateaddress                    | version + model |                                        |
+//! | verifymessage                      | returns boolean |                                        |
```

Comment:
You've added a type for this so this should be 'version'. (Adding the type is correct and the most thorough solution)

Reply:
This is not fixed yet mate. We need to propogate all these table changes upto v28 also. The `verify` tool is really letting us down here, it should have a way to catch this.